### ISSUE TITLE
Making text box bigger?

### ISSUE THREAD

**Issue by cusackjm** [episphere/connect#809]
Is it possible to make text boxes bigger, without the user having to expand the text box? This is in Module 1 (WORK3). I'm able to click on the lines in the bottom right corner to pull the text box down, but would like for the textbox to automatically be as wide as the text.
![image](https://github.com/episphere/connect/assets/85582895/ec7bdf62-aa56-49f4-a3c8-dfeb24357bf6)

![image](https://github.com/episphere/connect/assets/85582895/6a3ff217-fa3a-42ca-b5f7-ec60d4da7a16)

This is not an issue now, but will be an issue in the participant experience survey, which has long-answer open-ended questions.

**Comment by danielruss** [comment]
Yes, they can be made wider.  They may become wider when you don't expect them too.  I will make a change to Style1.css to widen the textarea to 95% of the parent container, which is the same as the buttons for multiple choice error.

**Comment by sonyekere** [comment]
@cusackjm Hi, can this issue be closed?

**Comment by cusackjm** [comment]
DEV: 
![image](https://github.com/episphere/connect/assets/85582895/4ba7876e-b1a9-41bb-b6e1-721ffa0ff7d3)

Will close this issue if things look different in stage, but otherwise textbox does not show all of the text 

@sonyekere @danielruss

**Comment by danielruss** [comment]
is this still a problem?

**Comment by m-j-horner** [comment]
UX issue is resolved in latest release of Baseline Modules into Quest 2.0
